spec/theme/plugins.spec.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+describe(\Dxw\MyTheme\Theme\Plugins::class, function () {
+    beforeEach(function () {
+        \WP_Mock::setUp();
+        \WP_Mock::wpFunction('esc_html', [
+            'return' => function ($a) {
+                return '_'.$a.'_';
+            },
+        ]);
+        $this->abspath = \org\bovigo\vfs\vfsStream::setup()->url() . '/';
+        $this->plugins = new \Dxw\MyTheme\Theme\Plugins(
```

Comment:
This looks like it might be an example of the 'mystery guest' smell: http://xunitpatterns.com/Obscure%20Test.html#Mystery%20Guest 

Basically in individual tests it makes it more difficult to see where data is coming from.

How sad would you feel about inlining this into some or all of the tests?

Another option would be to define a function which returns this object, which would let us specify a list of plugins in tests where we care about them, and not specify them in tests where we don't. Unfortunately I seem to recall that Peridot won't let you do define functions which are available in tests like that 😞